src/compiler/utilities.ts

```diff
@@ -230,9 +230,18 @@ namespace ts {
         return getSourceTextOfNodeFromSourceFile(getSourceFileOfNode(node), node, includeTrivia);
     }
 
+    function prefixWithUnderscoreUnderscore(identifier: string): boolean {

```

Comment:
if you don't like long expression in original code i'd rather add a local variable instead of a function